besu-plugins/linea-sequencer/sequencer/src/main/java/net/consensys/linea/credible/BlockEnv.java

```diff
@@ -1,263 +0,0 @@
-package net.consensys.linea.credible;
```

Comment:
Why are we removing this file?

Reply:
Moved all models to `SidecarApiModels.java`, seemed more clean